src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -92,30 +94,39 @@ const DisplayNote = React.forwardRef<HTMLElement, DisplayNoteProps>((
 
   useFocusElement(element, shouldFocusCard);
 
-  return <div className={className} ref={mergeRefs(ref, element)} tabIndex={-1} data-highlight-card>
-    <Dropdown toggle={<MenuToggle />} onToggle={onToggle} transparentTab={false}>
-      <DropdownList>
-        <DropdownItem message='i18n:highlighting:dropdown:edit' onClick={onEdit} />
-        <DropdownItem
-          message='i18n:highlighting:dropdown:delete'
-          data-testid='delete'
-          onClick={() => setConfirmingDelete(true)}
-        />
-      </DropdownList>
-    </Dropdown>
-    <CloseIcon onClick={onBlur} />
-    <label>Note:</label>
-    <TruncatedText text={note} isActive={isActive} onChange={() => setTextToggle((state) => !state)} />
-    {confirmingDelete && <Confirmation
-      ref={confirmationRef}
-      data-analytics-label='delete'
-      data-analytics-region='confirm-delete-inline-highlight'
-      message='i18n:highlighting:confirmation:delete-both'
-      confirmMessage='i18n:highlighting:button:delete'
-      onConfirm={onRemove}
-      onCancel={() => setConfirmingDelete(false)}
-    />}
-  </div>;
+  return (
+    <div
+      className={className}
+      ref={mergeRefs(ref, element)}
+      tabIndex={-1}
+      data-highlight-card
+      role='dialog'
+      aria-label={formatMessage({id: 'i18n:highlighter:display-note:label'})}
```

Comment:
Should this reflect the state of the card? This will show "highlight editor" for a non-editable state, which feels off to me. Maybe we should have "Highlighted note" for this and put "Highlight editor" on EditCard?

Reply:
Changed the message file and added one for the EditCard